exposee-app/server.js

```diff
@@ -12,9 +12,19 @@ import userRoutes from './Routes/users.js';
 import SequelizeStoreInit from 'connect-session-sequelize';
 import { DATE } from 'sequelize';
 import validate_Token from "./authentoken.js";
-//import { Video } from './models/video.js';
+
+import dotenv from 'dotenv';
+
 
 const app = express();
+dotenv.config();
+import Mux from '@mux/mux-node';
+const { Video: MuxVideo }  = new Mux(process.env.MUX_TOKEN_ID, process.env.MUX_TOKEN_SECRET);
+
+await MuxVideo.LiveStreams.create({
```

Comment:
so this is fine, but you need to use this in the endpoint. Your postman test never actually hits this code, so this code is "untested"

For your test, it should hit the /broadcast endpoint, which will then call this code you have here, and get a response which it uses to in the creation of the video model.